subsys/net/lib/fota_download/Kconfig

```diff
@@ -56,6 +56,12 @@ config FOTA_DOWNLOAD_RESOURCE_LOCATOR_LENGTH
 	int "Size of buffer used for Resource locator"
 	default 512
 
+config FOTA_DOWNLOAD_SEC_TAG_LIST_SIZE
```

Comment:
Maybe slap on a '_MAX' to indicate that this is not the actual size but rather the maximum allowed.